src/diffusers/models/unet_2d_blocks.py

```diff
@@ -531,11 +591,16 @@ def __init__(
         dual_cross_attention=False,
         use_linear_projection=False,
         only_cross_attention=False,
+        downsample_type: str = "default",
+        attention_block: str = "Transformer2DModel",
```

Comment:
That's too hacky, let's add a new downblock please

Reply:
done!